enzyme/Enzyme/CMakeLists.txt

```diff
@@ -208,6 +208,13 @@ target_compile_options(ClangEnzymeFlags INTERFACE "-fplugin=$<TARGET_FILE:ClangE
 add_custom_target(ClangEnzymeDummy "" DEPENDS ClangEnzyme-${LLVM_VERSION_MAJOR})
 add_dependencies(ClangEnzymeFlags ClangEnzymeDummy)
 
+set_property(TARGET ClangEnzyme-${LLVM_VERSION_MAJOR}
+    PROPERTY PUBLIC_HEADER
```

Comment:
can we use a glob here to not need changing if another file is changed?

Reply:
Sure